diff --git a/node_modules/mineflayer/lib/plugins/game.js b/node_modules/mineflayer/lib/plugins/game.js
index 12f54a4..a0fd1e0 100644
--- a/node_modules/mineflayer/lib/plugins/game.js
+++ b/node_modules/mineflayer/lib/plugins/game.js
@@ -94,7 +94,7 @@ function inject (bot, options) {
   })
 
   bot._client.on('game_state_change', (packet) => {
-    if (packet?.reason === 4 && packet?.gameMode === 1) {
+    if (packet?.reason === 4) {
       bot._client.write('client_command', { action: 0 })
     }
     if (packet.reason === 3) {
diff --git a/node_modules/mineflayer/lib/plugins/generic_place.js b/node_modules/mineflayer/lib/plugins/generic_place.js
index 8fddeba..5d7932b 100644
--- a/node_modules/mineflayer/lib/plugins/generic_place.js
+++ b/node_modules/mineflayer/lib/plugins/generic_place.js
@@ -34,7 +34,8 @@ function inject (bot) {
       dz = options.delta.z
     }
     if (options.forceLook !== 'ignore') {
-      await bot.lookAt(referenceBlock.position.offset(dx, dy, dz), options.forceLook)
+      // Skip looking for performance reasons
+      // await bot.lookAt(referenceBlock.position.offset(dx, dy, dz), options.forceLook)
     }
     // TODO: tell the server that we are sneaking while doing this
     const pos = referenceBlock.position
diff --git a/node_modules/mineflayer/lib/plugins/inventory.js b/node_modules/mineflayer/lib/plugins/inventory.js
index 85e545e..d98ab2e 100644
--- a/node_modules/mineflayer/lib/plugins/inventory.js
+++ b/node_modules/mineflayer/lib/plugins/inventory.js
@@ -155,7 +155,8 @@ function inject (bot, { hideErrors }) {
     const directionNum = vectorToDirection(direction) // The packet needs a number as the direction
     cursorPos = cursorPos ?? new Vec3(0.5, 0.5, 0.5)
     // TODO: tell the server that we are not sneaking while doing this
-    await bot.lookAt(block.position.offset(0.5, 0.5, 0.5), false)
+    // Skip looking for performance reasons
+    // await bot.lookAt(block.position.offset(0.5, 0.5, 0.5), false)
     // place block message
     if (bot.supportFeature('blockPlaceHasHeldItem')) {
       bot._client.write('block_place', {
diff --git a/node_modules/mineflayer/lib/plugins/physics.js b/node_modules/mineflayer/lib/plugins/physics.js
index a6dd8c8..b4b84f8 100644
--- a/node_modules/mineflayer/lib/plugins/physics.js
+++ b/node_modules/mineflayer/lib/plugins/physics.js
@@ -229,11 +229,8 @@ function inject (bot, { physicsEnabled }) {
     }
     lookingTask = createTask()
 
-    // this is done to bypass certain anticheat checks that detect the player's sensitivity
-    // by calculating the gcd of how much they move the mouse each tick
-    const sensitivity = conv.fromNotchianPitch(0.15) // this is equal to 100% sensitivity in vanilla
-    const yawChange = Math.round((yaw - bot.entity.yaw) / sensitivity) * sensitivity
-    const pitchChange = Math.round((pitch - bot.entity.pitch) / sensitivity) * sensitivity
+    const yawChange = yaw - bot.entity.yaw
+    const pitchChange = pitch - bot.entity.pitch
 
     if (yawChange === 0 && pitchChange === 0) {
       return
