diff --git a/node_modules/prismarine-nbt/nbt.js b/node_modules/prismarine-nbt/nbt.js
index ed46dfd..f024e10 100644
--- a/node_modules/prismarine-nbt/nbt.js
+++ b/node_modules/prismarine-nbt/nbt.js
@@ -231,7 +231,7 @@ const builder = {
   short (value) { return { type: 'short', value } },
   byte (value) { return { type: 'byte', value } },
   string (value) { return { type: 'string', value } },
-  comp (value, name = '') { return { type: 'compound', name, value } },
+  comp (value) { return { type: 'compound', value } },
   int (value) { return { type: 'int', value } },
   float (value) { return { type: 'float', value } },
   double (value) { return { type: 'double', value } },
